api/envoy/extensions/filters/http/jwt_authn/v3/config.proto

```diff
@@ -211,6 +211,13 @@ message JwtProvider {
   // Specify the clock skew in seconds when verifying JWT time constraint,
   // such as `exp`, and `nbf`. If not specified, default is 60 seconds.
   uint32 clock_skew_seconds = 10;
+
+  // This field only applies to :ref:`forward_payload_header <envoy_v3_api_field_extensions.filters.http.jwt_authn.v3.JwtProvider.forward_payload_header>`.
+  //
+  // When it is specified, the base64 encoded payload will be added to the
+  // header. Normally JWT based64 encode doesn't add padding. If this field is
+  // true, the padding will be added to the header.
+  bool add_padding_in_forward_payload_header = 11;
```

Comment:
Maybe change the name to `pad_forward_payload_header`?

Reply:
Done.